src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import Logo from "../icons/Logo.js";
+
+const Title = styled.header`
+  height: 80px;
+  width: 375px;
```

Comment:
Are you sure you want to use a fixed width for the parent container?

Reply:
In the project I wont use a fixed width, I just used it here to see how it looks like